src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,43 @@
-import React from 'react';
+import { TodoCompletedCategory } from '../../types/todoCompletedCategory';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { filterSlice } from '../../features/filter';
+
+const nameForCategoriesOption = {
+  [TodoCompletedCategory.all]: 'All',
+  [TodoCompletedCategory.active]: 'Active',
+  [TodoCompletedCategory.competed]: 'Competed',
```

Comment:
There is a typo in the key `TodoCompletedCategory.competed`. It should be `TodoCompletedCategory.completed` to match the expected enum value.